scripts/publish-charts.sh

```diff
@@ -7,22 +7,23 @@ TOOLS_DIR="${BUILD_DIR}/tools"
 STABLE="${GIT_REPO_ROOT}/stable"
 PACKAGE_DIR="${GIT_REPO_ROOT}/build"
 export PATH="${TOOLS_DIR}:${PATH}"
+VERSION="$(git describe --tags --always --dirty)"
 
-if echo "${CIRCLE_TAG}" | grep -Eq "^v[0-9]+(\.[0-9]+){2}$"; then
-    REPOSITORY="https://eks-bot:${GITHUB_TOKEN}@github.com/aws/eks-charts.git"
+if grep -Eq "^v[0-9]+(\.[0-9]+){2}$" <<< $VERSION; then
```

Comment:
Any reason for preferring - grep '-Eq` with `<<<` over `echo "${VERSION}" |grep` ?